### ISSUE TITLE
BaklavaJS V2 Save/Load graph

### ISSUE THREAD

**Issue by EugineKh** [newcat/baklavajs#224]
Hello.
Maybe I'm doing something wrong, but I had a problem loading the graph. After loading, the graph is drawn, but the links between the nodes are not redrawn after they are moved. Even if you delete links and create again, it will still not be attached to the nodes.

Saving and loading goes like this:
```
<script setup>
import { ref } from "vue"
import { EditorComponent, useBaklava } from "@baklavajs/renderer-vue"
import NodeStart from "./components/NodeStart"
import NodeMessage from "./components/NodeMessage"

const saveResults = ref('')

const baklava = useBaklava()
baklava.editor.registerNodeType(NodeStart)
baklava.editor.registerNodeType(NodeMessage)

function save() {
  saveResults.value = JSON.stringify(baklava.editor.save())
}

function load() {
  baklava.editor.load(JSON.parse(saveResults.value))
}
</script>

<template>
<button
  @click="save()">Save</button>
<button
  @click="load()">Load</button>
<div style="height: 100vh; width: 80vw">
  <EditorComponent :view-model="baklava" />
</div>
</template>
```

![BaklavaJS_V2](https://user-images.githubusercontent.com/90271793/202125091-f00ccd6b-c6d6-482c-9c3b-1a403ae6a857.gif)

**Comment by MioAtSwivel** [comment]
Can confirm on this, similar behavior also happen when moving the origin node, and the connection updates only when the destination node moves

**Comment by newcat** [comment]
Can you check whether this is fixed with [v2.0.2-beta.1](https://github.com/newcat/baklavajs/releases/tag/v2.0.2-beta.1), please?

**Comment by EugineKh** [comment]
Updated BaklavaJS to version v2.0.2-beta.1 and the problem disappeared. Now after loading the saved graph, everything is drawn as it should. Thank you so much!
